cmd/openshift/operator/kodata/openshift/tekton-results-route-rbac/rbac.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 The Tekton Authors
```

Comment:
@khrm please move this file to `cmd/openshift/operator/kodata/static/tekton-results/route-rbac.yaml`

Reply:
Sure.

Reply:
Moved to cmd/openshift/operator/kodata/static/tekton-results/route-rbac/rbac.yaml